src/compiler/scala/tools/nsc/backend/jvm/BCodeSkelBuilder.scala

```diff
@@ -390,8 +390,12 @@ abstract class BCodeSkelBuilder extends BCodeHelpers {
       }
 
       def load(locSym: Symbol) {
-        val Local(tk, _, idx, _) = slots(locSym)
-        bc.load(idx, tk)
+        slots.get(locSym) match {
+          case Some(Local(tk, _, idx, _)) =>
+            bc.load(idx, tk)
+          case None =>
+            ???
+        }
```

Comment:
- [ ] will do. this was needed to debug a failure in test/files/pos/SI-7638.scala, which I fixed by centralizing the removal of the symbol attachment from the cloned symbol of mixed in methods.